mysql-test/suite/tokudb.backup/t/tokudb_backup_disable_if_innodb_use_native_aio_enabled.test

```diff
@@ -0,0 +1,20 @@
+# Check if tokudb hot backup is prevented if innodb_use_native_aio enabled
+--source include/have_tokudb_backup.inc
+
+SELECT @@innodb_use_native_aio;
+
+--let BACKUP_DIR=$MYSQL_TMP_DIR/tokudb_backup
+
+--mkdir $BACKUP_DIR
+
+--replace_result $MYSQL_TMP_DIR MYSQL_TMP_DIR
+--error ER_WRONG_VALUE_FOR_VAR
+--eval SET SESSION tokudb_backup_dir='$BACKUP_DIR'
+
+--let $restart_parameters= restart:--innodb-use-native-aio=off
```

Comment:
Maybe not required? This is basic backup functionality, other tests in this suite already exercise it